lambda/reporting.tf

```diff
@@ -11,12 +11,13 @@ resource "aws_lambda_function" "reporting_lambda_function" {
   tags                           = var.common_tags
   environment {
     variables = {
-      AUTH_URL            = var.auth_url
-      CONSIGNMENT_API_URL = var.api_url
-      CLIENT_ID           = var.keycloak_reporting_client_id
-      CLIENT_SECRET       = aws_kms_ciphertext.environment_vars_reporting["client_secret"].ciphertext_blob
-      CLIENT_SECRET_PATH  = var.reporting_client_secret_path
-      SLACK_BOT_TOKEN     = aws_kms_ciphertext.environment_vars_reporting["slack_bot_token"].ciphertext_blob
+      AUTH_URL                 = var.auth_url
+      CONSIGNMENT_API_URL      = var.api_url
+      CLIENT_ID                = var.keycloak_reporting_client_id
+      CLIENT_SECRET            = aws_kms_ciphertext.environment_vars_reporting["client_secret"].ciphertext_blob
+      CLIENT_SECRET_PATH       = var.reporting_client_secret_path
+      SLACK_BOT_TOKEN          = aws_kms_ciphertext.environment_vars_reporting["slack_bot_token"].ciphertext_blob
+      TDR_REPORTING_CHANNEL_ID = aws_kms_ciphertext.environment_vars_reporting["tdr_reporting_channel_id"].ciphertext_blob
```

Comment:
Maybe rename this `TDR_REPORTING_SLACK_CHANNEL_ID` to make clear it's a slack channel id